packages/auth/src/access/service.test.ts

```diff
@@ -23,8 +23,8 @@ describe('Access Service', (): void => {
   beforeAll(async (): Promise<void> => {
     deps = await initIocContainer(Config)
     accessService = await deps.use('accessService')
-    knex = await deps.use('knex')
     appContainer = await createTestApp(deps)
+    knex = appContainer.knex
```

Comment:
```suggestion
```
`let knex: Knex` can be removed above and the `afterEach` below can be changed to
```
    await truncateTables(appContainer.knex)
```
I'm guessing this can happen in most of these test files

Reply:
It can happen in all these test files, but for some of them `knex` is used for querying.
```
Model.query(knex)
```
We keep the `knex` definition at the top for these test files or we change to `Model.query(appContainer.knex)`?

Reply:
weak vote for `Model.query(appContainer.knex)`